app/src/main/java/proofcastlabs/tee/security/Strongbox.java

```diff
@@ -383,12 +385,8 @@ public String getCertificateAttestation() {
             CBORFactory cborFactory = new CBORFactory();
             ObjectMapper mapper = new ObjectMapper(cborFactory);
             byte[] data = mapper.writeValueAsBytes(attestationCertificate);
-            ByteArrayOutputStream ba = new ByteArrayOutputStream();
-            ba.write(data);
 
-            return Base64
-                    .encodeToString(ba.toByteArray(), Base64.DEFAULT)
-                    .replaceAll("\n", "");
+            return new String(Hex.encodeHex(data));
```

Comment:
Shall we prefix it with `0x`?

Reply:
I don't have a preference, I checked other usages of `encodeHex` in the repo and none of them seem to be adding `0x` explicitly. Up to you!

Reply:
I'd add it, yeah

Reply:
Done!